drivers/l3gxxxx/Kconfig

```diff
@@ -0,0 +1,456 @@
+        of the output data is about Full Scale/INT16_MAX.
+
+config L3GXXXX_SCALE_245_DPS
+    bool "±245 dps"
+    help
+        The typical sensitivity is 8.75 mdps.
+
+if !MODULE_A3G4250D
+
+config L3GXXXX_SCALE_500_DPS
+    bool "±500 dps"
+    help
+        The Typical sensitivity is 17.5 mdps.
+
+config L3GXXXX_SCALE_2000_DPS
+    bool "±2000 dps"
+    help
+        The typical sensitivity is 70 mdps.
+
+endif # !MODULE_A3G4250D
+
+endchoice
+
+choice
+    bool "Filters used for output data"
+    default L3GXXXX_HPF_AND_LPF2
+    help
+        L3Gxxxx sensors integrate a combination of two low pass filters (LPF)
+        and one high pass filter (HPF) which can be used to filter the output
+        data. First, raw sensor data are always filtered by LPF1 with a cutoff
+        frequency that is fixed for the selected output data rate (ODR).
+        The resulting data can then optionally be filtered by HPF and LPF2.
+        Both filters can be used or bypassed.
+        Please note: The filter selection for the output data also affects the
+        filter selection for event interrupt generation. If the HPF is enabled
+        for filtering the output data, it is also active for filtering the
+        sensor data used for interrupt generation if the LPF2 is enabled for
+        interrupt generation.
+
+config L3GXXXX_NO_FILTER
+    bool "HPF and LPF2 are not used"
+
+config L3GXXXX_HPF_ONLY
+    bool "only HPF is used"
+
+config L3GXXXX_LPF2_ONLY
+    bool "only LPF2 used"
+
+config L3GXXXX_HPF_AND_LPF2
+    bool "HPF and LPF2 are used"
+
+endchoice
+
+if L3GXXXX_HPF_ONLY || L3GXXXX_HPF_AND_LPF2 || L3GXXXX_INT1_HPF_ONLY || L3GXXXX_INT1_HPF_AND_LPF2
+
+choice
+    bool "HPF mode"
+    help
+        The high pass filter (HPF) can be used in different modes. Select
+        the right one.
+
+config L3GXXXX_HPF_NORMAL
+    bool "Normal mode"
+    help
+        In Normal mode, the HPF is reset by reading REFERENCE register.
+
+config L3GXXXX_HPF_REFERENCE
+    bool "Reference mode"
+    help
+        In Reference mode, the output data are the difference to the
+        value written to the REFERENCE register.
+
+config L3GXXXX_HPF_AUTORESET
+    bool "Autoreset mode"
+    help
+        In Autoreset mode, the HPF is automatically reset when a configured
+        interrupt event occurs.
+
+endchoice
+
+config L3GXXXX_HPF_CUTOFF
+    int "HPF cutoff frequency n = [0...9]"
+    range 0 9
+    default 0
+    help
+        The cutoff frequency is specified as a value from 0 to 9, which defines
+        the cutoff frequency depending on the selected sensor mode and the
+        output data rate. Details can be found in the data sheet for the CTRL2
+        register field HPCF.
+
+endif # L3GXXXX_HPF_ONLY || L3GXXXX_HPF_AND_LPF2 || L3GXXXX_INT1_HPF_ONLY || L3GXXXX_INT1_HPF_AND_LPF2
+
+config MODULE_L3GXXXX_FIFO
+    bool "Support for the 32 level FIFO"
+    depends on MODULE_L3GXXXX
```

Comment:
As all this part is already inside `if MODULE_L3GXXXX` the `depends on` is redundant, right?

Reply:
right.